resources/data/elephpants.json

```diff
@@ -359,7 +359,14 @@
             "sponsor": "PHP Conference Poland",
             "year": 2019,
             "image": "https://i.imgur.com/QIyckVs.jpg"
+        },
+        {
+            "id": 46,
+            "name": "CakeDC Fest",
+            "description": "CakeDC Fest",
+            "sponsor": "CakePHP",
+            "year": 2017,
+            "image": "https://live.staticflickr.com/65535/48796213323_1efccd304c_n.jpg"
```

Comment:
please crop the image as a square to get the best shoot for the elePHPant. our command to add elePHPants will fit it in a square from the center, and I guess it's gonna cut his head :D

Reply:
This photo isn't the CakeFest elePHPant